pages/checkout/index.js

```diff
@@ -450,6 +450,13 @@ class CheckoutPage extends Component {
                               {item.product_name}
                             </p>
                             <p className="font-color-light">Quantity: {item.quantity}</p>
+                            <div className="d-flex justify-content-between mb-2">
+                              {item.variants.map((variant, i) =>
+                                <p key={i} className="font-color-light font-weight-small">
```

Comment:
```suggestion
                              {item.variants.map((variant) =>
                                <p key={variant.variant_id} className="font-color-light font-weight-small">
```

It's better to use some real data (e.g. the variant ID here) rather than a numeric array index where possible. React normally warns you when you do that.